tfx/components/example_gen/utils_test.py

```diff
@@ -285,9 +311,47 @@ def testSpanMatches(self):
         example_gen_pb2.Input.Split(name='s1', pattern='span{SPAN}/split1/*'),
         example_gen_pb2.Input.Split(name='s2', pattern='span{SPAN}/split2/*')
     ]
-    _, span = utils.calculate_splits_fingerprint_and_span(
+    _, span, _ = utils.calculate_splits_fp_span_and_version(
         self._input_base_path, splits)
     self.assertEqual(span, '02')
+  
+  def testVersionMatches(self):
```

Comment:
consider just have just one test for testing calculate_splits_fp_span_and_version function (instead of testSpan and testVersion) to reduce some duplication